static/sass/styles.scss

```diff
@@ -678,5 +685,65 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
 }
 
 .is-ticked.is-play-button::before {
-  @include vf-icon-video-play($color-mid-dark);
+  @include vf-icon-video-play($color-mid-dark); 
+}
+
+// XXX: pad content so it doesn't get to close to text in the adjacent column. Idea borrowed from the guardian. Propose this to vanilla.
+.u-pad--right {
+  padding-right: $sph--x-large;
+}
+
+.u-align-social-icons-to-text {
+  vertical-align: text-bottom;
 }
+
+.l-article {
+  @extend %vf-row;
```

Comment:
You simply extend `row` but then nest them later which causes you to override the expected padding of a row. Unexpected or difficult to explain events is a sign of something going in the wrong direction. 

If what you want to do is position the aside then let's step back and consider the entire page structure instead of this.

Reply:
you're right, fixed.